mesonbuild/interpreter/type_checking.py

```diff
@@ -615,6 +609,7 @@ def _target_install_convertor(val: object) -> bool:
                 ('1.1.0', 'generated sources as positional "objects" arguments')
         },
     ),
+    KwargInfo('build_subdir', str, default='', since='1.7.0')
```

Comment:
`1.10.0`

Reply:
oops! missed one. I've rechecked with 'git diff master | grep since' and everything says 1.10.0 now.